e2e/apprunner/apprunner_suite_test.go

```diff
@@ -36,10 +38,14 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(err).NotTo(HaveOccurred())
 	appName = fmt.Sprintf("e2e-apprunner-%d", time.Now().Unix())
+	err = command.Run("aws", []string{"ssm", "put-parameter", "--name", "my-ssm-param", "--value", "abcd1234", "--type", "String", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
```

Comment:
```suggestion
	err = command.Run("aws", []string{"ssm", "put-parameter", "--name", "e2e-apprunner-" + appName, "--value", "abcd1234", "--type", "String", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
```
to make sure we have different secrets created for each e2e test.

Reply:
I guess it's probably fine keeping a deterministic name 🤔  it'd be difficult to inject this value into `front-end/manifest.yml` otherwise. What do you and @iamhopaul123 think about naming it `e2e-apprunner-ssm-param`?

Reply:
+1 to `e2e-apprunner-ssm-param`, yes it would be extra effort to inject that value into `front-end/manifest.yml`

Reply:
OK let's keep it for now.